### DISCUSSION TITLE
Game - like the face of UPBGE.

### DISCUSSION THREAD

**Discussion by PawAlldeller** [UPBGE/upbge#1845] [Ideas/Features for UPBGE 0.3+]
The game is like the face of UPBGE.

Hello dear!

I have an idea that could be a reference point for the promotion and demonstration of the game engine

If we are somehow concerned with game development, then it would be cool to have a full-fledged game on the engine. 

A game that could serve as a benchmark, the face of the engine and demonstrate its power and innovations, give free rein to creativity, modifications. 

Simplify the study of the entire technical process of creating games, from the idea, modeling, styling, demonstration, to going out into the world.  And of course, to contribute to the expansion of the gaming community and work as a full-fledged product that finances developers with a donation system - jewelry. 

The idea is "simple" - To develop a complete game, under Open Source licenses. With the modification function. 

As an example, you can make an analogue of popular games such as CS, Dota 2, Fortnite, etc. 

The advantages of this approach is that you can use, in the future, an expandable community - third-party developers, enthusiasts, content creators to solve content filling issues, and other game developments. Thus, the community can participate in the development of the official game engine without the participation of the whole team, you only need an official plan of requirements and those.developments. 

Of course, with the addition of the name of the enthusiast in the credits =)

**Comment by BluePrintRandom** [comment]
Have you heard of project Wrectified? 
that has always been my goal for the project.
